INSTALLATION.md

```diff
@@ -183,6 +205,40 @@ This guide is for `VSCode` users to easily manage their `mongoDB` databases:-
 
 <br/>
 
+### Instructions to edit records for ADMIN user
+
+**Note**: You can skip these instructions for now if you don't have running instance of Talawa-Admin.
+
+#### I. MongoDB Compass
+
+1. Open a terminal and run `mongod` command to the start the MongoDB server.
+2. Open MongoDB Compass and click on `Connect`.
+3. Select `user` collections and edit the data. Change:
+     1. `userType` from ADMIN to SUPERADMIN
+     2. `adminApproved` from false to true
+     
+#### II. Mongo Shell
+
+1. Open a terminal and run `mongod` command to start the MongoDB server.
```

Comment:
This in not the case with Linux. Mongo automatically is started when installed or the system reboots. Do you need to run this command specifically for Windows, and if so why? It needs to be stated.

Reply:
@palisadoes Sir you're right I figured out that using mongod command is not always compulsory though it provides some diagnostic functionalities and also it might not be consistent  across all operating systems. We can omit this instruction it will not make any difference.